src/i18n/fi.json

```diff
@@ -88,6 +88,34 @@
     "approvedMessage": "Kiitos kun varmistit nuorisopalveluiden jäsenyyden.",
     "approvedLink": "Hakemus on jo hyväksytty / linkki on viallinen"
   },
+  "footer": {
+    "copyright": "Tekijänoikeus 2020",
```

Comment:
I wonder if 2020 should instead be the current year from a date?

Reply:
Changed to current year.